docs/docs/recipes.md

```diff
@@ -152,6 +152,51 @@ By targeting the HTML `body` element, your font will apply to most text on the p
 - More on [importing assets into files](/docs/importing-assets-into-files/]
 - [Using Typography.js for Google fonts](/docs/typography-js/)
 
+### Using Google Fonts
+
+#### Prerequisites
+
+- A [Gatsby site](/docs/quick-start)
+- The [Gatsby CLI](/docs/gatsby-cli/) installed
+- Choose a font package from [the typefaces project](https://github.com/KyleAMathews/typefaces)
+
+#### Directions
+
+1. Visit [the typefaces project](https://github.com/KyleAMathews/typefaces) and choose the font you'd like.
+2. Run `npm install --save yourchosenfont`, replacing `yourchosenfont` with the name of the font you want to install.
+3. Add `require("yourchosenfont")` to your site's entry file.
```

Comment:
We typically recommend ES6 imports these days, and there's more than one place to put it:
```suggestion
2. Add `import "yourchosenfont"` to a layout template, page component, or `gatsby-browser.js`.
```